transport/src/main/java/org/dst/drpc/api/Channel.java

```diff
@@ -0,0 +1,54 @@
+package org.dst.drpc.api;
+
+/**
+ * 一个链接，或者说一个可通信的信道
+ */
+public interface Channel {
+
+  /**
+   * 检查一个信道是否打开。
+   */
+//  boolean isOpen();
+//
+//  void open();
+//
+//  void close();
+
+  void send(Object message);
+
+//  void receive(Object message);
+
+
+
+  /**
+   * has attribute.
+   *
+   * @param key key.
```

Comment:
I don't think it's necessary to add these comments. The semantics of method names are very clear